no-computer.md

```diff
@@ -1,69 +1,60 @@
-# No computer project
+# No computer project: Badminton doubles hand-signals
 
 
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+This section describes what the no-computer DSL is and who uses it
 
+The fastest sport in the world. The only time badminton doubles players drop from that intensity is to serve, and it's one of the most important aspects of doubles. It dictates which side will have the advantage from the start, and the key part to serving well is communication between partners. But they can't just talk to each other, othewise the opponents would know exactly what they are about to do. Enter the DSL: hand signs behind the back. 
+
+The serving player stands in the front and uses these hand signs to let the partner at the back know which specific "play" they will use. While it's not nearly as complicated as a play in football or basketball - as there are only two players on each side - it is much more subtle.  
 
 ### What's the need?
-_What need is met by your idea? Who are you helping? What is that person's
-experience like now? What would their experience be like if you could help 
+_What need is met by this DSL? Who benefits? What would that person's experience be like without it? What would their experience be like with this DSL?
 them?_
 
+What do players gain from knowing exactly what serve his/her partner will use? In doubles badminton, the player serves from the frontcourt usually to the opponent's frontcourt to prevent the opponent from getting the upper hard (being able to attack with a smash). But occasionally, trick serves are used to catch the opponent off guard or to force an error. The most common one is a lob serve, which looks just like a normal serve, except at the last moment when the server flicks it just over the head of the opponent. This forces the opponent to have to prepare for both a high and a low shot. If he/she can anticipate this shot then he/she can easily jump and smash, intercepting the shuttle at an early and high point. Rally over. 
+
+The general rule in doubles is that when you lift (hit a high shot), partners move from the front/back position to a side/side position to cover as much area of the court as possible and allow for more reaction time. This is why it is so important to signal to your partner where you intend to serve so that the team can be well-prepared for the rally without losing the element of surprise. 
+
+Without these important hand-signs your partner could be surprised and you may easily lose the rally. 
```

Comment:
This seems to imply that the hand signals already exist?  Do they already exist and serve an important function in competitive badminton, or is this from your perspective after the adaptation of your new method?